src/components/App.js

```diff
@@ -15,49 +15,31 @@ class App extends Component {
     }
   }
 
-componentDidMount = () => {
-  this.setState({ isLoading: true} )
-  // fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-  //   .then(response => {
-  //     if(!response.ok) {
-  //       throw new Error("Oopsies! Something went wrong 🤡")
-  //     } else {
-  //       return response.json()
-  //     }
-  //   })
-  getMovies()
-    .then(data => this.setState({ movies: data.movies, isLoading: false }))
-    .catch(error => {
-      this.setState({ error: "ERROR: " + error.message })
-    })
-}
-
-  // findMovie = (id) => {
-  //   fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
-  //     .then(response => response.json())
-  //     .then(data => this.setState({ movie: data.movie }))
-  //     .catch(error => {
-  //       this.setState({ error: "ERROR: " + error.message })
-  //     })
-  // }
+  componentDidMount = () => {
+    this.setState({ isLoading: true })
+    getMovies()
+      .then(data => this.setState({ movies: data.movies, isLoading: false }))
+      .catch(error => {
+        this.setState({ error: "ERROR: " + error.message })
+      })
+  }
 
   render() {
     return (
       <div className='app'>
         <header className='siteHeader'>
-          <h1 className='siteTitle'> 🍿 Rancid Tomatillos 🍿</h1>
+          <h1 className='siteTitle'>🍿 Rancid Tomatillos 🍿</h1>
         </header>
-        {/* <SearchBar movies={this.state.movies} /> */}
-        { this.state.isLoading && <p>⏳ Loading...</p> }
-        { this.state.error && <p>{this.state.error}</p> }
+        {this.state.isLoading && <p className='loading'>⏳ Loading...</p>}
+        {this.state.error && <p className='error'>{this.state.error}</p>}
```

Comment:
I like how we made sure the user can definitely see the messages against the now black background ✔️